### PR TITLE
optimize Suse 12 support

### PR DESCRIPTION
**Pull request by ofsole**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -21,10 +21,19 @@
   # Check the client os to define the package name and service name
 
   case $::osfamily {
-    'Debian', 'Suse': {
+    'Debian': {
       $package_name = 'cron'
       $service_name = 'cron'
     }
+    'Suse': {
+      if $::lsbmajdistrelease == '12' {
```
Comment (Phil-Friderici) [inline manifests/init.pp]: I would try to not use the lsb*-facts. They need extra packages to be installed which might not be available on al systems.

Better use $::operatingsystemrelease and regex to match the major version:
```if $::operatingsystemrelease =~ /^12\./```

**Comment by Phil-Friderici** [comment]
Looks good, thanks for your work so far!

**Comment by ofsole** [comment]
yes, using operatingsystemrelease looks better for me :+1:

**Comment by Phil-Friderici** [comment]
I'll help fixing these Pv4 related errors.

**Comment by Phil-Friderici** [comment]
@ofsole I sent you a fix as PR.
Btw: you should create branches instead of working in master branch directly. This makes things much easier.

**Comment by Phil-Friderici** [comment]
Think it's easier to create a new PR https://github.com/Ericsson/puppet-module-cron/pull/18 containing our commits.

**Comment by ofsole** [comment]
you're right, i should avoid using maste-)r branch in my side.  i was considering the puppetv4 issue just now, thanks for your quick response. :-)

**Comment by ofsole** [comment]
then i will close this PR.

**Comment by Phil-Friderici** [comment]
@ofsole Thanks ! released in v2.0.0

**Pull request closed without merging**